checker/src/main/java/org/checkerframework/checker/interning/InterningAnnotatedTypeFactory.java

```diff
@@ -151,6 +152,16 @@ public Void visitDeclared(AnnotatedDeclaredType t, Void p) {
 
             return super.visitDeclared(t, p);
         }
+
+        @Override
+        public Void visitExecutable(AnnotatedExecutableType type, Void p) {
+            scan(type.getReturnType(), p);
+            // TODO: don't skip the receiver
```

Comment:
This will be fixed when I implement the new semantics for the Interning Checker.

Reply:
Shouldn't this be `TODO: skip the receiver`? Or am I missing the receiver here?

Reply:
D'oh. Misread this.